website/docs/d/iam_trusted_profiles.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile"
+description: |-
+  Get information about iam_trusted_profile
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile
+
+Provides a read-only data source for iam_trusted_profile. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information about trusted profile, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-profile
+
+## Example usage
+
+```terraform
+data "ibm_iam_trusted_profile" "iam_trusted_profile" {
+	profile_id = "profile_id"
+}
+```
+
+## Argument reference
+
+Review the argument reference that you can specify for your data source.
+
+* `profile_id` - (Required, Forces new resource, String) ID of the trusted profile to get.
+
+## Attribute reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the `iam_trusted_profile`.
+
+* `account_id` - (Required, String) ID of the account that this trusted profile belong to.
+
+* `created_at` - (Optional, String) If set contains a date time string of the creation date in ISO format.
+
+* `crn` - (Required, String) Cloud Resource Name of the item. For example: 'crn:v1:bluemix:public:iam-identity:us-south:a/myaccount::profile:Profile-94497d0d-2ac3-41bf-a993-a49d1b14627c'.
```

Comment:
Use the single quote as code tag:

'crn:v1:bluemix:public:iam-identity:us-south:a/myaccount::profile:Profile-94497d0d-2ac3-41bf-a993-a49d1b14627c'.

as

`crn:v1:bluemix:public:iam-identity:us-south:a/myaccount::profile:Profile-94497d0d-2ac3-41bf-a993-a49d1b14627c`

Note, in all the examples, use code tag instead **`** of single quote **'**